edweiss-app/components/courses/AssignmentDisplay.tsx

```diff
@@ -92,41 +92,4 @@ const AssignmentDisplay: ReactComponent<{ item: AssignmentWithColor, index: numb
 
 export default AssignmentDisplay;
 
-
-
-async function saveTodo(name: string, dueDate: Timestamp, description: string) {
-    try {
-
-
-        const res = await callFunction(Todolist.Functions.createTodo, {
-            name, description, dueDate: OurTime.toDate(dueDate).toISOString(), status: "yet"
-        });
-
-        if (res.status) {
-            Toast.show({
-                type: 'success',
-                text1: t(`course:toast_added_to_todo_text1`),
-                text2: t(`course:toast_added_to_todo_text2`)
-            });
-        } else if (res.error === "duplicate_todo") {
-            Toast.show({
-                type: 'info',
-                text1: t(`todo:already_existing_todo_toast_title`),
-                text2: t(`todo:already_existing_todo_toast_funny`)
-            });
-        } else {
-            Toast.show({
-                type: 'error',
-                text1: t(`todo:error_toast_title`),
-                text2: t(`todo:couldnot_save_toast`)
-            });
-        }
-    } catch (error) {
-        Toast.show({
-            type: 'error',
-            text1: t(`todo:error_toast_title`),
-            text2: t(`todo:couldnot_save_toast`)
-        });
-    }
-
-}
\ No newline at end of file
+export async function saveTodo(name: string, dueDate: Timestamp, description: string) { try { const res = await callFunction(Todolist.Functions.createTodo, { name, description, dueDate: OurTime.toDate(dueDate).toISOString(), status: "yet" }); if (res.status) { router.back(); Toast.show({ type: 'success', text1: t(`course:toast_added_to_todo_text1`), text2: t(`course:toast_added_to_todo_text2`) }); } else { Toast.show({ type: 'error', text1: t(`todo:error_toast_title`), text2: t(`todo:couldnot_save_toast`) }); } } catch (error) { Toast.show({ type: 'error', text1: t(`todo:error_toast_title`), text2: t(`todo:couldnot_save_toast`) }); } }
```

Comment:
Jamming it all in one line is unreadable, is it okay if you can let it breath a little ?

Reply:
Yes of course